src/Traits/TokenAwareTrait.php

```diff
@@ -40,8 +40,9 @@ public function getToken(): Token
      */
     public function getRequestHeaders(): array
     {
+        $tokenType = ucfirst(strtolower($this->token->getTokenType()));
```

Comment:
I added this after changing the tests for the endpoint.
Generally, the "Bearer" string is case insensitive, however, people advise to keep it `Ucfirst`, as some HTTP libraries tend to be more strict. This seems reasonable also for our tests.

Reply:
ok 4 me